python/pyarrow/pandas_compat.py

```diff
@@ -36,6 +36,17 @@
 from pyarrow.lib import _pandas_api, frombytes  # noqa
 
 
+try:
+    _np_unicode = np.unicode_
+except AttributeError:
+    _np_unicode = np.str_
+
+try:
+    _np_sctypes = np.sctypes
+except AttributeError:
+    from numpy.core.numerictypes import sctypes as _np_sctypes
```

Comment:
Thank you for contributing, much appreciated!

One question, is the reason for the Try Except older Python version (or numpy?) or can we just replace `np.unicode_` with `np.str_` and `np.sctypes` with `sctypes` (like `np.NaN` with `np.nan`)?

Reply:
I'm just trying out a run locally with numpy 1.16.6